secretmanager/api/SecretManagerSample/Program.cs

```diff
@@ -42,6 +42,9 @@ class SecretOptions
 
         [Value(1, HelpText = "ID of the secret", Required = true)]
         public string SecretId { get; set; }
+
+	[Value(2, HelpText = "Etag of the secret", Required = false)]
```

Comment:
Hey @gwhitehawk - I think there may be some formatting issues throughout. Can you run `dotnet format` or use your editor plugin to format the code?

Reply:
Oh oh, just used vim with lacking vimrc, fixed now.